homeassistant/components/nextdns/config_flow.py

```diff
@@ -128,3 +128,31 @@ async def async_step_reauth_confirm(
             data_schema=AUTH_SCHEMA,
             errors=errors,
         )
+
+    async def async_step_reconfigure(
```

Comment:
Don't we need to have a check here that the profile exists with this new key?
Probably the same question for `reauth` too.